Matrix.py

```diff
@@ -0,0 +1,123 @@
+class Matrix:
+    def __init__(self, x):
+        if type(x) is tuple:
+            for tup in x:
+                if len(tup) == len(x) and type(tup) is tuple:
+                    self.x = x
+                else:
+                    self.x = None
+                    print("invalid Matrix")
+        else:
+            self.x = None
+            print("invalid matrix")
+
+    def __iter__(self):
+        return iter(self.x)
+
+    def __getitem__(self, item):
+        return self.x[item]
+
+    def __repr__(self):
+        string = ""
```

Comment:
its a bad habit use variables names such as string.

Reply:
changed to "matrix_in_lines"